vector/src/main/java/im/vector/app/features/widgets/WidgetActivity.kt

```diff
@@ -168,7 +173,7 @@ class WidgetActivity : VectorBaseActivity<ActivityWidgetBinding>() {
     private fun createElementCallPipParams(): PictureInPictureParams? {
         val actions = mutableListOf<RemoteAction>()
         val intent = Intent(ACTION_MEDIA_CONTROL).putExtra(EXTRA_CONTROL_TYPE, CONTROL_TYPE_HANGUP)
-        val pendingIntent = PendingIntent.getBroadcast(this, REQUEST_CODE_HANGUP, intent, 0)
+        val pendingIntent = PendingIntent.getBroadcast(this, REQUEST_CODE_HANGUP, intent, FLAG_IMMUTABLE)
```

Comment:
This isn't strictly related to the intent of this PR but the widget kept crashing for me without this.

Reply:
Yes it seems mandatory started from Android 12: https://developer.android.com/about/versions/12/behavior-changes-12#pending-intent-mutability